airflow/config_templates/config.yml

```diff
@@ -1562,6 +1562,41 @@
       type: string
       example: ~
       default: "512"
+    - name: use_row_level_locking
+      description: |
+        Should the scheduler issue `SELECT ... FOR UPDATE` in relevant queries.
+        If this is set to False then you should not run more than a single
+        scheduler at once
+      version_added: 2.0.0
+      type: boolean
+      example: ~
+      default: "True"
+    - name: max_dagruns_to_create_per_loop
+      description: |
+        This changes the number of dags that are locked by each scheduler when
+        creating dag runs. One possible reason for setting this lower is if you
+        have huge dags and are running multiple schedules, you won't want one
+        scheduler to do all the work.
+
+        Default: 10
+      example: ~
+      version_added: 2.0.0
+      type: string
+      default: ~
+    - name: max_dagruns_per_loop_to_schedule
+      description: |
+        How many DagRuns should a scheduler exaimne (and lock) when scheduling
+        and queuing tasks. Increasing this limit will allow more throughput for
+        smaller DAGs but will likely slow down throughput for larger (>500
+        tasks for example.) DAGs. Setting this to high when using multiple
```

Comment:
```suggestion
        tasks for example) DAGs. Setting this too high when using multiple
```

Reply:
:face_with_head_bandage: Working late never pays off. I'll fix this in the morning :laughing:

Reply:
I know it 👌

Reply:
Done in https://github.com/apache/airflow/pull/10956/commits/70fd17fc697a688545dd523ed5b91e74e7d81149